.travis.yml

```diff
@@ -2,3 +2,9 @@ language: dart
 dart: dev
 script: ./tool/travis.sh
 sudo: false
+branches:
+  only:
+    - master
```

Comment:
This is a misleading line. It's not what you think it means. :)

Traditionally, Travis will kick off two builds when you push a PR, or update a PR: one build for the branch and one build for the PR. This is duplicate work.

This config line here implicitly says "run a test when you push a PR, and when master changes"